build/teamcity-testrace.sh

```diff
@@ -19,7 +19,7 @@ if [[ "${TC_BUILD_BRANCH}" == master ]] || [[ "${TC_BUILD_BRANCH}" == release* ]
 	PKGSPEC="./pkg/..."
 else
 	git fetch origin master
-	PKGSPEC=$(git diff --name-only $(git merge-base HEAD origin/master) | grep "^pkg*.go" || true)
+	PKGSPEC=$(git diff --name-only $(git merge-base HEAD origin/master) | grep "^pkg.*\.go" || true)
```

Comment:
I know this already landed, but I'll plug `git diff master...` (or `git diff master...HEAD`) as an equivalent and useful shorthand.

Also, no need for grep; `git diff` can limit itself to a directory:

```
git diff --name-only master... -- pkg
```

Reply:
To be exactly equivalent, you probably want: `git diff --name-only master...  -- 'pkg/**/*.go'`